builder/builder.go

```diff
@@ -1331,37 +1358,37 @@ func (b *Builder) BuildBundles(template *x509.Certificate, privkey *rsa.PrivateK
 	defer timer.WriteSummary(os.Stdout)
 
 	timer.Start("BUILD BUNDLES")
+	var addLocalConf bool
 	if _, err := os.Stat(b.DNFConf); os.IsNotExist(err) {
-		outfile, err := os.Create(b.DNFConf)
+		if err = ioutil.WriteFile(b.DNFConf, []byte(fmt.Sprintf(dnfConfFormat, b.UpstreamURL)), 0644); err != nil {
```

Comment:
Optional: This line is really long. I would put `if err != nil {` on the next line just to make it easier to read.